cli/replicaset/cartridge.go

```diff
@@ -175,6 +180,17 @@ func (c *CartridgeInstance) Expel(ctx ExpelCtx) error {
 	return newErrExpelByInstanceNotSupported(OrchestratorCartridge)
 }
 
+// BootstrapVShard bootstraps the vshard for a single instance by the Cartridge orchestrator.
+func (c *CartridgeInstance) BootstrapVShard(ctx VShardBootstrapCtx) error {
+	err := cartridgeBootstrapVShard(c.evaler, ctx.Timeout)
+	cartridgeLogVShardBoostrapInfo(err)
```

Comment:
Why we log something only here, but not for other subcommands?

Reply:
It's the command specific: https://github.com/tarantool/cartridge-cli/blob/master/cli/replicasets/bootstrap_vshard.go#L28-L44

Reply:
So why not just return it as an error message?

Reply:
Yes, we can, did this, the error is wrapped.